src/attribute/attribute.service.ts

```diff
@@ -1,9 +1,27 @@
 import { Injectable } from '@nestjs/common';
 import { PrismaService } from 'src/prisma/prisma.service';
+import { CreateAttributeDto } from './dto/attribute.dto';
 
 @Injectable()
 export class AttributeService {
-  constructor(private prisma: PrismaService) {}
+  constructor(private prisma: PrismaService) { }
+
+  async create(createAttributeDto: CreateAttributeDto) {
+    return await this.prisma.attribute.create({
```

Comment:
Returning a response here but in controller that response is not being handled. Refactor the code with proper validations